src/content/docs/zh-cn/reference/errors/csssyntax-error.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: CSS Syntax Error
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **错误信息示例:**<br/>
+CSSSyntaxError: 缺少分号<br/>
+CSSSyntaxError: 未闭合的字符串<br/>
```

Comment:
I don't think there's any need for a translation here.

Reply:
Okay, I'll make some modifications.